site/content/docs/user/configuration.md

```diff
@@ -161,6 +161,18 @@ nodes:
 - role: worker
 {{< /codeFromInline >}}
 
+You can also set a specific Kubernetes version by setting the node image. You can find available versions on [DockerHub](https://hub.docker.com/r/kindest/node/tags):
```

Comment:
Please don't recommend this. We don't currently guarantee that those tags will be compatible. You should check the release notes for a list of images we shipped with the release OR build a custom image

Reply:
Okay, can we instead of not describing how to do this, update the verbiage? It seems disadvantageous to not document this.

Reply:
right, we should document setting image, just not doing so by checking docker hub tags, please point to the release notes instead and recommend including the `sha256` bit.

Reply:
we should also give an example following best practices I.E., including the full documented image.

Reply:
I'm not sure what you had in mind. I'm not super familiar with `kind` so I wouldn't consider myself authoritative on best practices.